src/main/java/org/takes/rs/RsWithType.java

```diff
@@ -47,10 +52,12 @@
     public RsWithType(final Response res, final CharSequence type) {
         super(
             new RsWithHeader(
-                new RsWithStatus(res, HttpURLConnection.HTTP_OK),
-                "Content-Type", type
+                new RsWithoutHeader(
+                    new RsWithStatus(res, HttpURLConnection.HTTP_OK),
+                    HEADER
+            ),
```

Comment:
@carlosmiranda Here I must say I agree with you but I had to change this as it is now to get rid of a static analysis error. So, the change is ok IMHO.